websvc.saas.codegen.java/src/org/netbeans/modules/websvc/saas/codegen/java/resources/apikey.properties

```diff
@@ -1,3 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I wonder, since this is part of the `codegen` package, does the user get to see this? Because if so, we should also exclude it. If not, it's OK.

Can you check?